crates/sui-framework/packages/move-stdlib/sources/vector.move

```diff
@@ -158,6 +158,40 @@ module std::vector {
         v.pop_back()
     }
 
+    /// Take all elements of the vector `v` except the first `n` elements. Aborts if `n` is greater
+    /// than the length of `v`. Modifies the original vector.
+    public fun skip_mut<T>(v: &mut vector<T>, n: u64): vector<T> {
+        assert!(n <= v.length());
```

Comment:
I think this is the correct choice, but just wanted to double check. 
If folks don't want it to assert they should do `let len = v.length(); v.skip(n.min(len))`, which hopefully makes this more explicit for the person auditing/reading the code